iothub/device/tests/TransportSettingsTests.cs

```diff
@@ -146,14 +144,37 @@ public void AmqpTransportSettings_TimeoutPropertiesSet()
         }
 
         [TestMethod]
-        public void AmqpTransportSettings_TimeoutDefaultsAndOverrides()
+        public void AmqpTransportSettings_TimeoutDefaults()
```

Comment:
I would rename this to AmqpTransportSettings_SetsDefaultTimeout

Reply:
I can do that, sure!